packages/schemas/tables/users.sql

```diff
@@ -8,6 +8,8 @@ create table users (
   password_encrypted varchar(128),
   password_encryption_method password_encryption_method,
   password_encryption_salt varchar(128),
+  name varchar(128),
+  avatar varchar(256),
```

Comment:
nit: as `avatar` here is an URL, is there any case that the length of `avatar` exceeds the length limit of 256?